index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="style.css" />
+  <title>Document</title>
+</head>
+
+<body>
+  <nav id="navbar">
+    <a href="#profile">profile</a>
+    <a href="#about">about me</a>
+    <a href="#contact">contact</a>
+  </nav>
+  <br />
+  <br />
+  <section id="profile">
+    <h1>Taman balkhi</h1>
+    <h2>Software Developer</h2>
+    <p>
+      I can help you build a product , feature or website Look through some of
+      my work and experience! If you like what you see and have a project you
+      need coded, don not hestiate to contact me.
+    </p>
+  </section>
+  <div class="skills" id="about">
+    <div class="languages">
+      <ul>
+        <li class="list">Language</li>
+        <li>Javascript</li>
+        <li>Html</li>
+        <li>Css</li>
+      </ul>
+      <ul>
+        <li class="list">Frameworks</li>
+        <li>Bootstrap</li>
+      </ul>
+      <ul>
+        <li class="list">Skills</li>
+        <li>Github</li>
+        <li>Gitlab</li>
+        <li>Terminal</li>
+      </ul>
+    </div>
+  </div>
+  <br>
+  <br>
+  <br>
+  <br>
+
+  <section id="contact">
+    <h2>contact me</h2>
+    <br />
+    <br />
+    <a href="https://www.facebook.com/tamana.saiedy">facebook</a>
+    <a href="https://github.com/tamana-Balkhi">github</a>
+    <a href="#0093788130271">whatsapp</a>
+  </section>
+</body>

```

Comment:
Hi, your design still does not match the Figma template of your choosing. Kindly fix it. 
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/176984789-4a6b83ae-0fed-4885-982c-7b8ccacbf33e.png) | ![image](https://user-images.githubusercontent.com/33268927/176984857-a9836d70-22cb-4be4-aa9b-760ac525005e.png) |